sale_zero_stock/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': "Sale Order Approval",
+    'depends': ['sale'],
+    'category': 'Sales/Sales',
+    'summary' : "Approval for sale order",
+    'version': '1.0',
+    'description': "This module checks for approval before confirming Sale Order",
```

Comment:
```suggestion
    'description': """
This module checks for approval before confirming Sale Order
""",
```
Reference: https://www.odoo.com/documentation/18.0/contributing/documentation/rst_guidelines.html#contributing-rst-doc-hyperlinks